parastell/parastell.py

```diff
@@ -14,7 +15,7 @@
 from .utils import read_yaml_config, filter_kwargs, m2cm
 
 build_cubit_model_allowed_kwargs = ["skip_imprint", "legacy_faceting"]
-export_dagmc_allowed_kwargs = [
+export_cf_dagmc_allowed_kwargs = [
```

Comment:
What does `cf` mean here?

Reply:
I think it's for Coreform?  I wonder if `cubit` would be better than `cf`?

Reply:
Yes, `cf` refers to Coreform. I was copying their DAGMC export syntax in older versions of Cubit, but I agree using `cubit` would be clearer in our case.